C/plugins/common/omf.cpp

```diff
@@ -1569,3 +1651,24 @@ bool OMF::getCreatedTypes(const string& key)
 	}
 	return ret;
 }
+
+/**
+ * Check whether input Datapoint type is supported by OMF class
+ *
+ * @param    dataPoint		Input data
+ * @return			True is fupported, false otherwise
+ */ 
+
+static bool isTypeSupported(DatapointValue& dataPoint)
```

Comment:
Please make this a private method of the class.

Reply:
It's used by two classes.